src/EnergyPlus/WaterCoils.cc

```diff
@@ -1992,41 +1945,150 @@ namespace WaterCoils {
                 WaterConvectTerm = 0.0;
             }
             if ((AirConvectTerm > 0.0) && (WaterConvectTerm > 0.0)) {
-                WaterCoil(CoilNum).UACoilVariable = 1.0 / ((1.0 / WaterConvectTerm) + WaterCoil(CoilNum).FoulingFactor + (1.0 / AirConvectTerm));
+                WaterCoil(CoilNum).UACoilVariable = 1.0 / ((1.0 / WaterConvectTerm) + (1.0 / AirConvectTerm));
             } else {
                 // use nominal UA since variable UA cannot be calculated
-                WaterCoil(CoilNum).UACoilVariable = 1.0 / (1.0 / WaterCoil(CoilNum).UACoil + WaterCoil(CoilNum).FoulingFactor);
+                WaterCoil(CoilNum).UACoilVariable = WaterCoil(CoilNum).UACoil;
+            }
+
+            // calculate the Faulty Coil Fouling (thermal insulance) Factor using fault information
+            if (WaterCoil(CoilNum).FaultyCoilFoulingFlag &&
```

Comment:
@jmarrec @JasonGlazer @Myoldmopar I'm a bit confused about what this `FaultyCoilFoulingFactor` means. I would have just reported a ratio of `UACoilVariable` (fouled) to `OriginalUACoilVariable`, but everyone else seems to be ok with this, so go ahead.

Reply:
@mjwitte thanks, I think we should for now proceed as-is and get this one behind us.

@jmarrec if anything needs to change here, open up an issue and continue addressing things there, for next release.  (Unless it's urgent, small, etc., etc., ...)